setup.cfg

```diff
@@ -1,2 +1,30 @@
 [metadata]
-description-file = README.md
\ No newline at end of file
+description-file = README.md
+[flake8]
+ignore=E501,F401,E128,E402,E731,F821,E712
+exclude =
```

Comment:
just use include here, 

```
include=masonite, tests
```

That way only masonite folder will be targeted.

I'm not sure, if we should add `tests` folder for pep8 checking, @josephmancuso your thoughts?